Introduce new templates

abangser opened the pull request:
I attempted to introduce a template for new contributions but in using the new format I added the wrong file extension. This is a format that requires `.yml` not `.md` therefore the template is not visible on the repo.

In addition, this commit also introduces another new template for translations. This is in an attempt to make it easier for people to add translations to the site. I have noticed that people are being great about building off past PRs, but I thought codifying it in a template could be helpful.

UPDATE:
This PR opened up a bunch of ideas around how to support contributors. This has resulted in a number of changes including using the contributing section of the website in addition to templates to (hopefully) enable people better! The idea is that we want to become more transparent with how people can get involved, and more likely that when they do present something it will be supported and successful.

lianmakesthings commented:
Thanks for this @abangser. I am a bit unclear how to use these templates though. Could you provide some guidance for this in the readme or contributing.md?

abangser commented:
Good point @lianmakesthings. I have pushed a few small updates:
1. I introduced a translations section under contribute with some of the content from the issue template plus a bit more
2. I updated the link in the getting started blog where I already referenced the community contributing template
3. I have added a push to open this type of issue on the contributing page of the site

I hope these help with what you were asking, but let me know if there are other ways to make it easier for people to get started!

abangser commented:
Any chance the updates helped @lianmakesthings ? (Or others!)

Thanks ☺️

lianmakesthings commented on website/content/en/contribute/_index.md:
```diff
@@ -25,6 +25,10 @@ There’s everything from code-specific issues to documentation issues and every
 
+**Contribute a new idea**
+
+While there are always a number of ongoing topics in the TAG, they all started as new ideas. If you have a new proposal we encourage you to [open an issue using the `community contribution` template](https://github.com/cncf/tag-app-delivery/issues/new) and start to build the community of people who would want to work with you on it!
```
Does this still need to be added on GH side? Or will this happen with merging this PR?
We could also link directly to [this wizardy form](https://github.com/abangser/tag-app-delivery/issues/new?assignees=&labels=proposals&projects=&template=community-contribution.yml&title=%5BCommunity%5D+%3CShort+description%3E) (which I now realise lives on your repo)

abangser replied:
This is only on my repo because I wanted to make it visible before we merged to main here. By merging my commits to main on my fork I am able to preview the changes before we need to merge them here. I am not sure of any other way to do the preview.

This was the lesson I learned from the first merge where everything _looked_ OK as yaml but still had errors once on main and actually applied to the GitHub Issues tab!

lianmakesthings replied:
Gotcha. Right now, if you click on the link, you just get to the New Issue page with no way to choose the "community contribution template".
This can be quite confusing. Is the expectation, that as soon as we merge the PR, it will be possible to choose the template from https://github.com/cncf/tag-app-delivery/issues/new?

lianmakesthings reviewed (changes requested):
Amazing work @abangser!
Can we just link to the two new issue types in the same way?
Either direct links for both or the way you explained it for the website translation issues
> To get started, please [open an issue](https://github.com/abangser/tag-app-delivery/issues/new/choose) on the TAG App Delivery GitHub repository using the `Website Translation` issue type.
